x/tokenfactory/client/cli/tx.go

```diff
@@ -199,3 +200,34 @@ func NewChangeAdminCmd() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// NewChangeAdminCmd broadcast MsgChangeAdmin
+func NewSetBeforeSendHook() *cobra.Command {
```

Comment:
any meaning of such a CLI command given the fact the only allowed sender is the adminmodule?

Reply:
Why you think so? It's okay if you can create a denom and as an admin you can set a before hook after.
probably you mean that only allowed sender of `MsgUpdateParams` is an admin module.

Reply:
probably. no idea now 🤡